src/main/resources/se/diabol/jenkins/pipeline/DeliveryPipelineView/RegExpSpec/help-showUpstream.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Will show upstream jobs for the first job.
```

Comment:
@KostyaSha the first job in the pipeline could have upstream dependencies that could trigger a new build of that job. See example screenshot above.